packages/core-react/src/components/Card/index.js

```diff
@@ -2,10 +2,34 @@ import React from 'react';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
 
-export default function Card({ heading, body, img, row }) {
+import rayExports from '@wework/ray-core/scss/ray-exports.scss';
+
+const ASPECT_RATIOS = (rayExports['ray-image-aspect-ratios'] || '').split(' ');
+
+export default function Card({
+  heading,
+  body,
+  img,
+  row,
+  aspectRatio,
+  className,
+  ...props
+}) {
   return (
-    <div className={clsx('ray-card', { 'ray-card--row': row })}>
-      <div className="ray-card__image ray-image ray-image--16by9">{img}</div>
+    <div
+      {...props}
+      className={clsx('ray-card', {
+        'ray-card--row': row,
+        [className]: className
+      })}
+    >
+      <div
+        className={clsx('ray-card__image', 'ray-image', {
+          [`ray-image--${aspectRatio}`]: !!aspectRatio
```

Comment:
Is the boolean coercion necessary? 
```suggestion
          [`ray-image--${aspectRatio}`]: aspectRatio
```